src/igvfd/audit/file_set.py

```diff
@@ -34,7 +34,15 @@ def audit_no_files(value, system):
     '''
     object_type = space_in_words(value['@type'][0]).capitalize()
     description = get_audit_description(audit_no_files)
-    if not (value.get('files', '')):
+    assay_term = value.get('assay_term', '')
+    if assay_term:
+        assay_term = system.get('request').embed(assay_term, '@@object?skip_calculated=true')
+        assay_term = assay_term.get('term_name', '')
+    # Measurement sets from these CRISPR assays do not expect any sequence data
+    CRISPR_assays = ['proliferation CRISPR screen',
+                     'CRISPR perturbation screen followed by flow cytometry and FISH']
+    preferred_assay_title = value.get('preferred_assay_title', '')
+    if not (value.get('files', '')) and assay_term not in CRISPR_assays and preferred_assay_title not in ['Variant FlowFISH']:
```

Comment:
should not we rely on Cas mediated mutagenesis instead? https://data.igvf.org/assay-terms/OBI_0003133/ why variant flowfish is the only one relying on preferred title?

Reply:
exactly as it appears in measurement set.py below

Reply:
I wasn't sure if other cas mediated mutagenesis screens would not expect to have files in the measurement set.